### PR TITLE
Updated sql cheat sheet

### PR DESCRIPTION
**Pull request by mohan08p**
DCL and TCL keys added(five keys)

IA page: https://duck.co/ia/view/sql_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gautamkrishnar** [comment]
LGTM :+1:  @mohan08p

File: share/goodie/cheat_sheets/json/sql.json [hunk 1]
```diff
@@ -8,13 +8,14 @@
    },
    "template_type":"reference",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/sql.json]: We should have a space after the colon, please fix this throughout the file.

**Comment by mohan08p** [comment]
thanks @GuiltyDolphin @moollaza @gautamkrishnar

**Comment by jdorweiler** [comment]
@mohan08p can you update your branch it has some conflicts.

**Comment by mohan08p** [comment]
@jdorweiler : thanks :+1:

File: share/goodie/cheat_sheets/json/sql.json [hunk 2]
```diff
@@ -1,236 +1,251 @@
...
+			"val": "Remove an index"
+		}, {
+			"key": "DROP VIEW",
+			"val": "Remove a view"
+		}, {
+			"key": "TRUNCATE TABLE",
+			"val": "Empty a table"
+		}],
+		"Data Manipulation Language (DML)": [
+			{
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/sql.json]: Spacing is a *bit* off here!

File: share/goodie/cheat_sheets/json/sql.json [hunk 3]
```diff
@@ -1,236 +1,251 @@
...
+		"SQL Constraints"
+	],
+	"aliases": [
+		"SQL Constraints",
+		"Scalar Functions"
+	],
+	"aliases": [
+		"sql statements",
+		"sql commands"
+	],
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/sql.json]: We seem to have duplicate aliasas!

File: share/goodie/cheat_sheets/json/sql.json [hunk 4]
```diff
@@ -1,236 +1,251 @@
...
-         {  
-            "key":"DEFAULT",
-            "val":"Specifies a default value for a column"
-         }
-      ]
-   }
-}
+{
+	"id": "sql_cheat_sheet",
+	"name": "SQL Statements",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/sql.json]: We should be using 4 spaces per indent - you seem to have converted to tabs (hence the large diff).

**Comment by jdorweiler** [comment]
:+1:  thanks, @mohan08p

**Pull request merged**